docs/tutorial/Create-the-bundle.md

```diff
@@ -1,75 +1,63 @@
+Once you have [installed eZ Platform](https://doc.ez.no/x/opPfAQ), including the creation of a database for the tutorial, [configured your server](https://doc.ez.no/pages/viewpage.action?pageId=31429536), and [started your web server](https://doc.ez.no/display/DEVELOPER/Web+Server), you need to create a code base for the tutorial.
 
-
-
-<span class="tL8wMe EMoHub">Once you have [installed eZ Platform](https://doc.ez.no/x/opPfAQ), including the creation of a database for the tutorial, [configured your server](https://doc.ez.no/pages/viewpage.action?pageId=31429536), and [started your web server](https://doc.ez.no/display/DEVELOPER/Web+Server), you need to create a code base for the tutorial. </span>
-
-<span class="tL8wMe EMoHub">We will use <a href="http://symfony.com/doc/current/cookbook/bundles/index.html" class="external-link">the Symfony 2 extension mechanism, bundles,</a> to wrap the Fieldtype. </span>You can get started with a bundle using the built-in Symfony 2 bundle generator, following the instructions on this page.
+We will use [the Symfony 2 extension mechanism, bundles,](http://symfony.com/doc/current/cookbook/bundles/index.html) to wrap the Fieldtype. You can get started with a bundle using the built-in Symfony 2 bundle generator, following the instructions on this page.
 Then you will configure your Bundle to be able to write the code you need to create a Field Type.
 
-<span class="aui-icon aui-icon-small aui-iconfont-info confluence-information-macro-icon"></span>
-The<a href="https://github.com/ezsystems/TweetFieldTypeBundle" class="external-link">tutorial Github repository</a> shows you the Bundle finished.
+The [tutorial's Github repository](https://github.com/ezsystems/TweetFieldTypeBundle) shows you the Bundle in a finished state.
 
-Generating the bundle
----------------------
+## Generating the bundle
 
 From the eZ Platform root, run the following:
 
-``` brush:
+``` bash
 php app/console generate:bundle
 ```
 
 First, you are asked:
 
-``` brush:
+``` bash
 Are you planning on sharing this bundle across multiple applications? [no]: yes <enter>
 ```
 
 Type the answer `yes` and submit it with an Enter.
 
 Next you will be asked about the namespace of your bundle.
 
-More about naming bundles
+#### More about naming bundles
 
-<span class="aui-icon aui-icon-small aui-iconfont-info confluence-information-macro-icon"></span>
-See <a href="http://symfony.com/doc/current/cookbook/bundles/best_practices.html#bundle-name" class="uri" class="external-link">http://symfony.com/doc/current/cookbook/bundles/best_practices.html#bundle-name</a> for more details on bundle naming conventions.
+See (http://symfony.com/doc/current/cookbook/bundles/best_practices.html#bundle-name) for more details on bundle naming conventions.
 
 Put **EzSystems\\TweetFieldTypeBundle** as Bundle namespace, then the name of the bundle will be hinted from this entry.
 
-**Namespace bundle**
-
-``` brush:
+``` bash
```

Comment:
`brush` is not a syntax highlighting recognized by github-flavoured markdown, it is an artifact left by Confluence